tests/test_viewdom_static_string.py

```diff
@@ -45,7 +45,7 @@ def test_child_nodes():
     result = html(t"<div>Hello <span>World<em>!</em></span></div>")
     assert str(result) == "<div>Hello <span>World<em>!</em></span></div>"
 
-
+@pytest.mark.skip
 def test_doctype():
     """Sometimes it is hard to get a DOCTYPE in to the resulting output."""
```

Comment:
it's not ... it's the `\n` that is useless so that the assertion will fail?

Reply:
Correct, I fixed this in the more-viewdom branch and didn't want to re-fix it here.

Reply:
current main has this fixed